### PR TITLE
Add prolif as mdakit entry

### PR DESCRIPTION
**Pull request by IAlibay**

### CODE REVIEW

**Comment by IAlibay** [comment]
@cbouy if you're happy with the metadata here, I think it's good to merge. We'll try to keep CI going so it should keep you updated on any breaking changes upstream.

File: mdakits/prolif/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  - pip install git+https://github.com/chemosim-lab/ProLIF@master
+python_requires: ">=3.8"
+mdanalysis_requires: ">=2.2.0"
+run_tests:
+  - git clone latest
+  - pytest -v ./tests
+test_dependencies:
+  - mamba install pytest
+orgname: chemosim-lab
+development_status: Beta
```
Comment (cbouy) [inline mdakits/prolif/metadata.yaml:32]: I guess it's `Production/Stable` now but I still have to update it in the setup.cfg classifiers

**Comment by cbouy** [comment]
Apart from a nitpicky comment LGTM 👍 

> We'll try to keep CI going so it should keep you updated on any breaking changes upstream.

Ooooh nice!

File: mdakits/prolif/metadata.yaml (hunk 1 above)
Comment (IAlibay) [reply mdakits/prolif/metadata.yaml]: 😅 Yeah I might have just copied stuff out of the pypi metadata

**Pull request merged**